src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -5,10 +5,9 @@
 import io.swagger.annotations.ApiParam;
```

Comment:
This shouldn't be here

Reply:
Please review again, this branch wasn't update with main.